dev/ci/README.md

```diff
@@ -71,24 +67,38 @@ When you submit a pull request (PR) on Coq GitHub repository, this will
 automatically launch a battery of CI tests. The PR will not be integrated
 unless these tests pass.
 
-Currently, we have two CI platforms:
+We are currently running tests on many different platforms:
```

Comment:
I feel that "many" is a bit overkill, why not just "We currently run tests on the following CI platforms:"

Reply:
OK